config/endpoints/14_dgfip_attestation_fiscale.yml

```diff
@@ -6,8 +6,8 @@
     entity_type_description: |+
       Cette API délivre uniquement les attestations fiscales des **✅ entreprises soumises à l'impôt sur les sociétés (IS)**. 
 
-      L'API ne permet pas de récupérer les attestations fiscales : 
-      - ❌ des entreprises qui relèvent de l'impôt sur le revenu (IR), les entreprises individuelles sont donc notamment exclues ;
+      **L'API ne permet pas de récupérer les attestations fiscales** : 
+      - ❌ des entreprises qui relèvent de l'impôt sur le revenu (IR), les entreprises individuelles et micro-entreprises sont donc exclues. _Les entreprises qui relèvent des BIC, BNC ou BA, bénéfices respectivement industriels ou commerciaux, non-commerciaux ou agricoles, sont imposées à l'impôt sur le revenu. Leur attestation fiscale n'est donc pas récupérable par cette API_ ;
```

Comment:
@Haelle est-ce que tu valides cette formulation ?

Reply:
Non, justement **de ma compréhension** les BIC, BNC ou BA ne relève pas le l'IR, ils peuvent relever de l'IR ; et donc dans certains car il relève de l'IS et donc on peut les obtenir, mais ces cas sont marginaux. 

Ce que je propose c'est d'ajouter cette subtilité en faisant comprendre que ça dépend de ce qu'à choisit l'exploitant. S'il a choisit d'imposer ses bénéfices sur son IR ou le faire via une société.

Reply:
### Ce n'est pas ce que j'ai compris de mes différentes recherches.

Je pose ici les notes de ce que j'ai compris : 
Il y a deux types d'imposition : 
- IR 
- IS

Ces deux types d'imposition, qui portent toutes deux sur des bénéfices, ont leurs propres régimes. BIC BA et BNC sont des régimes, toujours associé au mode d'imposition IR. 
Les bénéfices passibles de l'IS sont déterminés de la même manière ou presque que les les BIC, mais on ne parle jamais de BIC si le type d'imposition est l'IS.

**Source intéressante :** 
- [entreprendre.service-public.fr](https://entreprendre.service-public.fr/vosdroits/F32919) Dans un encart il est bien dit que si l'entrepreneur opte pour une assimilation à EURL, il est soumis à l'IS et _"ne ne relève plus des règles concernant l'imposition des bénéfices industriels et commerciaux"_
- [Un cours intéressant sur le droit](https://www.fallaitpasfairedudroit.fr/droit-fiscal/fiscalite-des-entreprises/bic-impot-sur-les-societes/332-les-principes-generaux-en-bic-impot-sur-les-societes-cours)

### Comme ça va être compliqué de trancher et que c'est super technique,
je vais faire une autre proposition moins clivante

Reply:
Ok je te fais confiance on peut partir sur ta dernière proposition